shared/constants/network.ts

```diff
@@ -643,6 +650,13 @@ export const CHAIN_ID_TO_CURRENCY_SYMBOL_MAP = {
     CHAINLIST_CURRENCY_SYMBOLS_MAP.ACALA_NETWORK,
 } as const;
 
+export const CHAIN_ID_TO_CURRENCY_SYMBOL_MAP_NETWORK_COLLISION = {
```

Comment:
Would be good to add a comment above this describing its purpose, so people know when to add to it in the future